File: tarea5/polinomio2.c
#include <stdio.h>

int main(int argc, char *argv[]) {
	
	
	
	int c[4]={2,5,3,-2};
	
	int e=1;
	int suma1;
	int suma;
	
	
	
	printf("%dx^%d + %d \n", c[0],e,c[1] );
	printf("+\n");	printf("%dx^%d  %d \n", c[2],e,c[3] );
	printf("____________\n");
	
	suma = c[0]+c[2];
	suma1= c[1]+c[3];	
printf("%dx^%d  + %d ",suma,e,suma1);
	
	return 0;
}

File: tarea5/polinomio2.scala

   object Polinomio2{

     def main(args: Array[String])
     {
       var c = Array(2,5,3,-2)
       var e = 1;
       var Suma1 = 0;
       var Sum = 0;

       println(""+c(0)+"x^"+e+ "+" +c(1))
       println("+")
       println(""+c(2)+"x^"+e+ "" +c(3))
       println("_________")

       Sum = c(0)+c(2)
       Suma1= c(1)+c(3)

       println(""+Sum+"x^"+e+"+"+Suma1)

     }


   }
